### PR TITLE
Custom root level queries and mutations via inheritance

### PR DESCRIPTION
**Pull request by ninabernick**
Add custom root level queries and mutations by inheriting the codegenned `Query` and `Mutation` classes. Also supports overriding codegenned resolvers. Adds docs on custom code.

### CODE REVIEW

**Comment by jgadling** [comment]
This is perfect, and so simple! 😍 

Just two things:
- I don't think we need to rename the base Query / Mutation classes to BaseQuery / BaseMutation - if someone wants to override them, they can come up with different class names
- Can you write this up into a HOWTO style document (maybe as a markdown file in a `docs` dir?) so we have examples of how to extend platformics in various ways?

File: docs/custom_code.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# How To: Extending Platformics Generated Code
+Platformics will generate a GraphQL API that exposes CRUD operations for each entity type in your LinkML schema. It is often the case, however, that you will need to expose more functionality in your API than just CRUD. There are a few options for extending generated code to add custom business logic to your application.
+
+## Adding root-level queries and mutations
+Platformics will generate basic read and aggregate queries in `api/queries.py`. These are inherited by a query class in `custom_queries.py`, which will gain access to all the codegenned queries as well as allow you to implement your own queries. To add a new root level field (query), add the field to the query class in `custom_queries.py`:
+
+```
```
Comment (j-x-han) [inline docs/custom_code.md:7]: nit: you can do ```python for python syntax highlighting!

Comment (ninabernick) [reply docs/custom_code.md:7]: ooh!

**Review by j-x-han — APPROVED** [review]
LGTM! 🎉

**Pull request merged**